es-vue-base/tests/__snapshots__/EsSlider.spec.js.snap

```diff
@@ -7,82 +7,90 @@ exports[`EsSlider <EsSlider
             :min="100"
             :max="500"
             ariaLabel="Select a number" /> 1`] = `
-"<div class="es-slider vue-slider vue-slider-ltr" style="padding: 14px 14px; width: auto; height: 4px;" min="100" max="500">
-  <div class="vue-slider-rail">
-    <div class="vue-slider-process" style="height: 100%; top: 0px; left: 0%; width: 50%; transition-property: width,left; transition-duration: 0.5s;"></div>
-    <div class="vue-slider-marks">
-      <div class="vue-slider-mark vue-slider-mark-active" style="height: 100%; width: 4px; left: 0%;">
-        <div class="vue-slider-mark-step vue-slider-mark-step-active"></div>
-        <div class="slider-label vue-slider-mark-label active">
-          100
+"<client-only min="100" max="500">
```

Comment:
Should this `<client-only>` element be output to the HTML?

Reply:
Good catch. This won't happen in the app, it's likely because `jest` isn't aware of what `client-only` is. I'll work on a stub.